modules/ReactBreakpoints.js

```diff
@@ -115,23 +115,17 @@ class ReactBreakpoints extends React.Component {
     }
   }
   calculateCurrentBreakpoint(screenWidth) {
-    let currentBreakpoint = null
-    const breakpointKeys = Object.keys(this.props.breakpoints)
-    new Array(...breakpointKeys)
-      .reverse() // reverse array to put largest breakpoint first
-      .map(breakpoint => {
-        const breakpointValue = this.props.breakpoints[breakpoint]
-        if (!currentBreakpoint && screenWidth >= breakpointValue) {
-          currentBreakpoint = breakpoint
-        }
-      })
-    // If currentBreakpoint is null here, screenWidth is below lowest breakpoint,
-    // so it will still be set to equal lowest breakpoint instead of null
-    if (currentBreakpoint === null) {
-      currentBreakpoint = breakpointKeys[0]
+    const breakpoints = Object.keys(this.props.breakpoints)
+      .map(k => [k, this.props.breakpoints[k]])
+      .sort((a, b) => a[1] < b[1])
```

Comment:
This would mean that the breakpoints object will get sorted every time `calculateCurrentBreakpoint` is run, which is on every `resize` event at the very least. I'm thinking that maybe it is a better idea performance wise to break the sorting out into a function and maybe run it on the breakpoint objects on the props received so it is only run once. We can run it again if `props.breakpoints` changes for some reason but we will have to be careful to not break the SSR functionality.

I don't know if it's easier to move this logic out now, or swallow the performance hit and fix this later.

What's your opinion on moving the sort out of `calculateCurrentBreakpoint` ?